.github/workflows/actions_build.yml

```
@@ -0,0 +1,145 @@
+name: CI
+on: [push, pull_request]
```

Could we apply push events only for the `master` branch?
```suggestion
on:
  push:
    branches:
      - master
  pull_request:
```

---

By the way, wonder if rewriting the CI is a good time to rename the branch to `main` from `master`

---

@trustin @ikhoon any thought on this? 🤔

---

It is a good time to change to the new default branch even though we manually update the upstream branch of all active PRs.

---

Nah active PRs are automatically updated now (ever since a few months ago). As far as I know the only annoyance now is updating workflows, which is happening right now anyways

---

@trustin has suggested using v1 for 1.X.Y and v2 for 2.X.Y etc. instead of `main` and `master` branches. Is it okay for you? 🤔